cmd/crc/cmd/daemon_windows.go

```diff
@@ -0,0 +1,9 @@
+package cmd
+
+import (
+	"net"
+)
+
+func httpListener() (net.Listener, error) {
+	return nil, nil
```

Comment:
log not supported? or not implemented here?

Reply:
I wanted to add named pipe in a separated commit for more clarity.